assets/app/styles/_core.less

```diff
@@ -203,6 +203,19 @@ html, body {
   }
 }
 
+.nav-tabs {
+  font-size: @font-size-base;
+  a {
+    // ui.bootstrap.tabs doesn't set the href attr, so the cursor style is
```

Comment:
we talked about wanting the active tab to be linkable, so we can go straight to the log from elsewhere in the UI, and so that browser history works when switching between tabs